modules/cassandra/src/test/java/org/testcontainers/containers/CassandraDriver3Test.java

```diff
@@ -31,7 +31,7 @@ public void testCassandraGetContactPoint() {
 
             KeyspaceMetadata keyspace = session.getMetadata().getKeyspaces().get(CqlIdentifier.fromCql("test"));
 
-            assertNotNull("Failed to create test keyspace", keyspace);
+            assertThat(keyspace).as("Failed to create test keyspace").isNotNull();
```

Comment:
```suggestion
            assertThat(keyspace).as("keyspace created").isNotNull();
```

Else this would be `withFailMessage`.